compiler/damlc/daml-ide-core/dlint.yaml

```diff
@@ -965,3 +976,8 @@
 # foo = typeOf (undefined :: a) -- typeRep (Proxy :: Proxy a)
 # {-# RULES "Id-fmap-id" forall (x :: Id a). fmap id x = x #-}
 # </TEST>
+
+# --
+# DAML specific
+
+- ignore: {name: Use newtype instead of data}
```

Comment:
As I wrote elsewhere, is there a way to turn this off by default and allow users to explicitly opt in here? Having to ignore a rule that doesn't make too much sense for DAML feels bad, at least for me.

Reply:
This line is turning it off. It can be re-enabled by users in their `.dlint.yaml`s if they want.